Webmecanik/plugin-grapesjs-builder
# GrapesJs as default builder

Requires this PR on Mautic side: https://github.com/mautic/mautic/pull/9935

#### Steps to test this PR:
1. Clone Mautic locally
2. cd to the plugins directory
2. Remove the GrapesJS plugin and clone this repo: https://github.com/mautic/plugin-grapesjs-builder into the folder `GrapesJsBuilderBundle` - can be done using `gh repo clone mautic/plugin-grapesjs-builder GrapesJsBuilderBundle` if you have GitHub CLI installed
3. cd into the `GrapesJsBuilderBundle` directory
3. Apply the PR https://github.com/mautic/plugin-grapesjs-builder/pull/79 - can be done using `gh pr checkout 79` if you have GitHub CLI installed
2. cd back to Mautic's root directory
2. Apply this PR https://github.com/mautic/mautic/pull/9935 - can be done using `gh pr checkout 9935` if you have GitHub CLI installed
3. GrapesJs should now be the default builder without additional changes being needed (enable/disable)...

GrapesJs plugin is not as integrations anymore. By default is loaded with related themes (no builder in config, or grapesjsbuilder in config)
Not need enable/disable plugin.  Any themes by default are open by GrapesJs If any builder integration is not enabled.
Should cleanup set of themes in M4 or do we include all current themes?
